src/scss/nl-design-system-community.scss

```diff
@@ -1,5 +1,4 @@
 @import '@utrecht/components/dist/document/css/index.css';
-@import '@utrecht/components/dist/form-field/css/index.css';
 @import '@utrecht/components/dist/form-field-description/css/index.css';
```

Comment:
this will probably need to be moved so that it's included in the prod bundle :warning: